src/colordetection.py

```diff
@@ -21,20 +22,24 @@ def get_color_name(self, hsv):
         """
         (h,s,v) = hsv
         #print((h,s,v))
-        if h < 15 and v < 100:
-            return 'red'
-        if h <= 10 and v > 100:
-            return 'orange'
-        elif h <= 30 and s <= 100:
+        if s <= color_params.sat_W and v >= color_params.val_W:
```

Comment:
I trust your logic more than mine, but are you 100% that you should use `<=` and `>=` ? Because my logic is `<` and `<`

Reply:
I am fairly sure that it is correct yes, but will run some tests tonight.